ts/store/actions/wallet/outcomeCode.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Action types and action creator related to OutcomeCodeState.
+ */
+import { Option } from "fp-ts/lib/Option";
+import { ActionType, createStandardAction } from "typesafe-actions";
+
+export const outcomeCodeRetrieved = createStandardAction(
```

Comment:
```suggestion
export const paymentOutcomeCode = createStandardAction(
  "PAYMENT_OUTCOME_CODE"
)<Option<string>>();
```
Could it be a better naming?

Reply:
Maybe should we provide an action to reset the store section `lastPaymentOutcomeCode` that update it to the initial state?

Reply:
Added in [47cf051](https://github.com/pagopa/io-app/pull/2842/commits/47cf051ccb41e1bcb6ec143c0cdd84d9c98efbfd)